cdm/src/main/java/thredds/crawlabledataset/utils/S3Helper.java

```diff
@@ -0,0 +1,121 @@
+package thredds.crawlabledataset.utils;
+
+import net.sf.ehcache.*;
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.model.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+
+public class S3Helper {
+    private static String S3_PREFIX = "s3://";
+    private static String S3_DELIMITER = "/";
+    private static HashMap<String, File> fileStore = new HashMap<String, File>();
```

Comment:
Probably should not be static since you are instantiating a specific helper.

Also, make it a concurrent hash map - hashmap is not thread safe and multiple threads will be adding files to this store